docs/specs/xref.yml

```
@@ -2162,3 +2162,69 @@ outputs:
   .errors.log: |
     {"message_severity":"warning","code":"missing-attribute","message":"Missing required attribute: 'name'.","file":"docs/TOC.md"}
     {"message_severity":"warning","code":"xref-not-found","message":"Cross reference not found: 'b'.","file":"docs/TOC.md"}
+---
+# files to be redirected must be excluded from xref map
+noSingleFile: true
+repos:
+  https://docs.com/redirected-uid-excluded-from-xrefa#master:
+    - files:
+        docfx.yml:
+        _themes/ContentTemplate/schemas/TestData.schema.json: |
```

Simplify this test case by assigning uids to markdowns:
```suggestion
       a.md: |
       ---
       uid: a
       ---
```